### ISSUE TITLE
Parent/child selectors

### ISSUE THREAD

**Issue by abb1991** [301gene/prj-rev-bwfs-tea-cozy#3]
I notice you use a lot of parent/child relationships to determine your elements like this:

```
header img {
  height: 50px;
  padding-left: 10px;
}

. . . 

nav span {
  display: inline-block;
  padding: 20px 10px;
}
```

While this is an effective strategy and a good use of a new concept, can you defend why you'd want to style your element this way rather than through a direct class/id?

**Comment by 301gene** [comment]
I have to agree that using a direct class/id is a better strategy in this case as there is only one container in the entire website that I am styling when using the aforementioned parent/child selector. Plus, an id/class would be more descriptive.
